src/sockperf.cpp

```diff
@@ -3346,6 +3371,24 @@ int bringup(const int *p_daemonize) {
                 rc = SOCKPERF_ERR_NO_MEMORY;
             } else {
                 memset(reinterpret_cast<void *>(tmp), 0, sizeof(struct fds_data));
+                if (s_user_params.addr.ss_family == AF_UNIX) {
+                    log_dbg("UNIX domain socket was provided %s\n", s_user_params.addr.addr_un.sun_path);
+                    s_user_params.tcp_nodelay = false;
+                    if (s_user_params.sock_type == SOCK_DGRAM) { // Need to bind localy
+                        s_user_params.client_bind_info.ss_family = AF_UNIX;
+                        if (strlen(s_user_params.client_bind_info.addr_un.sun_path) == 0 && s_user_params.mode == MODE_CLIENT) { // no specific addr client_info was provoided
```

Comment:
```suggestion
                        if (s_user_params.client_bind_info.addr_un.sun_path[0] == 0 && s_user_params.mode == MODE_CLIENT) { // no specific addr client_info was provoided
```

Reply:
"_client" was replaced in 'feedfile' commit for <PATH>_<PID>_<FD>

Reply:
my comment about strlen is still actual

Reply:
we can close this one, changed from `strlen` to `[0]`